utils/sparse.py

```diff
@@ -34,20 +50,31 @@ def __init__(self, model, recipe, checkpoint_recipe=None, checkpoint_epoch=float
         if self.checkpoint_recipe_manager:
             _LOGGER.info("Applying structure from checkpoint recipe")
             self.checkpoint_recipe_manager.apply_structure(
-            module=model,
-            epoch=checkpoint_epoch,
+                module=model,
+                epoch=checkpoint_epoch,
         )
 
     def state_dict(self):
+        """
+        :return: A dict object with compsed recipe
```

Comment:
Weird, somehow I cannot suggest changes in this PR.
Anyway typo: `compsed` -> `composed`